App/src/components/ScheduleModal.js

```diff
@@ -123,67 +128,70 @@ function ScheduleModal({
-            <ul css={listCss}>
-              {shiftsClicked.map(shift => (
-                <li>
-                  {nameFrontDesk(shift.id) +
-                    " of shift " +
-                    shift.shift_id +
-                    " of date " +
-                    shift.date}
-                </li>
-              ))}
-            </ul>
-            <div css={{ display: "flex", flexDirection: "column" }}>
-              <span css={{ fontWeight: "bold" }}> Motive: </span>
-              <textarea
-                css={textareaCss}
-                type="text"
-                value={newMotive}
-                onChange={handleChangeMotive}
-              />
-            </div>
-          </form>
-        </div>
-        <div css={footer}>
-          <button type="button" onClick={onRequestClear}>
-            Clear
-          </button>
-          <button type="button" onClick={handleCreateRequest}>
-            Send
-          </button>
+    <>
+      <Modal
+        isOpen={isOpen}
+        onRequestClose={onRequestClose}
+        style={customStyles}
+        contentLabel="Example Modal"
+        ariaHideApp={isOpen}
+      >
+        <div css={container}>
+          <div css={head}>
+            <h2>Schudule Change</h2>
+            <span
+              onClick={onClose}
+              css={cancel}
+              aria-label="Close schedule modal"
+            >
+              &times;
+            </span>
+          </div>
+          <Calendar
+            workShiftConcat={workShiftConcat}
+            frontdesks={frontdesks}
+            calcDay={calcDay}
+            start={start}
+            end={end}
+            shiftsClicked={shiftsClicked}
+            saveShiftsClicked={setShiftsClicked}
+          />
+          <div>
+            <form css={formCss}>
+              <ul css={listCss}>
+                {shiftsClicked.map(shift => (
+                  <li>
+                    {nameFrontDesk(shift.id) +
+                      " of shift " +
+                      shift.shift_id +
+                      " of date " +
+                      shift.date}
+                  </li>
+                ))}
+              </ul>
+              <div css={{ display: "flex", flexDirection: "column" }}>
+                <span css={{ fontWeight: "bold" }}> Motive: </span>
+                <textarea
+                  css={textareaCss}
+                  type="text"
+                  value={newMotive}
+                  onChange={handleChangeMotive}
+                />
+              </div>
+            </form>
+          </div>
+          <div css={footer}>
+            <button type="button" onClick={onRequestClear}>
+              Clear
+            </button>
+            <button type="button" onClick={handleCreateRequest}>
+              Send
+            </button>
+          </div>
         </div>
-      </div>
-    </Modal>
+      </Modal>
+      {alert !== "" && <Alert message={alert} onClose={() => setAlert("")} />}
```

Comment:
This condition is the same as just `alert`.
```suggestion
      {alert && <Alert message={alert} onClose={() => setAlert("")} />}
```

Reply:
Done. thanks